### ISSUE TITLE
Fix radial-gradient()

### ISSUE THREAD

**Issue by hinok** [Compass/compass#1284]
Hello,
I use radial-gradient function, so I have:

```
$experimental-support-for-svg: false;
@include background-image(radial-gradient(0 50%, ellipse cover, rgba(#000, 0.5) 0%, rgba(#000, 0) 75%));
```

it was converted to:

```
background-image: -webkit-gradient(radial, 0 50%, 0, 0 50%, 75, color-stop(0%, rgba(0, 0, 0, 0.5)), color-stop(75%, rgba(0, 0, 0, 0)));
background-image: -webkit-radial-gradient(0 50%, ellipse cover, rgba(0, 0, 0, 0.5) 0%, rgba(0, 0, 0, 0) 75%);
background-image: -moz-radial-gradient(0 50%, ellipse cover, rgba(0, 0, 0, 0.5) 0%, rgba(0, 0, 0, 0) 75%);
background-image: -o-radial-gradient(0 50%, ellipse cover, rgba(0, 0, 0, 0.5) 0%, rgba(0, 0, 0, 0) 75%);
background-image: radial-gradient(0 50%, ellipse cover, rgba(0, 0, 0, 0.5) 0%, rgba(0, 0, 0, 0) 75%);
```

Ok, this css **doesn't work** in latest Opera 12 :(

Official radial-gradient definition (without browser's prefix) has form:

```
radial-gradient(5em circle at top left, yellow, blue)
```

And this css work in latest Opera 12

```
background: radial-gradient(ellipse at 0 50%,  rgba(0,0,0,0.5) 0%,rgba(0,0,0,0) 75%);
```

@source
http://dev.w3.org/csswg/css-images-3/#radial-gradients
https://developer.mozilla.org/en-US/docs/Web/CSS/radial-gradient?redirectlocale=en-US&redirectslug=CSS%2Fradial-gradient

Compass 0.12.2 (Alnilam)
Mac OS X Snow Leopard

**Comment by gcapizzi** [comment]
Do you know which browser would use the definition without prefix? Do they support the syntax you propose?

**Comment by chriseppstein** [comment]
We are aware of this bug and are working on a fix.

**Comment by chriseppstein** [comment]
fixed by 3e03ec00a2e0bbd73623a7a97aa95b3975b12159
